dimod/sampleset.py

```diff
@@ -1534,7 +1539,7 @@ def aggregate(self):
         # fix the number of occurrences
         record.num_occurrences = 0
         for old_idx, new_idx in enumerate(inverse):
-            record[new_idx].num_occurrences += self.record[old_idx].num_occurrences
+            record["num_occurrences"][new_idx] += self.record[old_idx].num_occurrences
```

Comment:
I'm curious, what's the reason for this change?

Reply:
The old one doesn't work anymore. I think it may be a bug in NumPy? I couldn't find a release note or bug report though.